packages/openapi-jsonschema-parameters/test/data-driven/convert-a-single-nullable-complex-query-param-to-json-schema.js

```diff
@@ -60,11 +60,7 @@ module.exports = {
               uniqueItems: false,
               enum: ['1', '3'],
               multipleOf: 57,
-              examples: {
```

Comment:
* i'm a bit concerned that we're breaking the format here.  is this used internally anywhere else in open-api?
* out of curiosity, what caused you to notice this?
* can you provide a link to examples in the openapi v3 spec?

Reply:
Here are the links to examples in OpenAPI v3 spec.
- [In component object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#components-object)
- [In parameter obejct](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#fixed-fields-10)
- [In media type object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#media-type-object)

And here are the [example object itself](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#example-object) and the [JSON schema v7 spec](https://json-schema.org/understanding-json-schema/reference/generic.html#annotations).

Talking about the code you highlight, if I'm correct, `convert-a-single-nullable-complex-query-param-to-json-schema.js ` is the test data for OpenAPI v2 (swagger). I have read the specification of OpenAPI v2 and do not find any definition of `examples`, however, I'm not sure what will be the right strategy to convert the field mentioned in v3 but not mentioned in v2, so I make it stay in the original strategy.

This problem happened when I tried to add some examples in the parameter object in the main API spec which is named `apiDoc.yaml` in the official readme. The sample:
```yaml
openapi: 3.0.2
info:
  version: 1.0.0
  title: Title
components:
  parameters:
    sortQuery:
      in: query
      name: sort
      schema:
        type: string
      examples:
        single:
          value: key1
paths: {}
```

Reply:
ok.  thanks @IvanaGyro!  i'm comfortable merging this.